src/pal/src/configure.cmake

```diff
@@ -366,6 +366,9 @@ check_cxx_source_runs("
 #include <stdlib.h>
 #include <time.h>
 #include <sys/time.h>
+#ifdef (__linux__)
```

Comment:
The parentheses are syntax error - they cannot be used with #ifdef. It can only be followed by a symbol.
What do you mean by "the `__LINUX__` is not valid"? As I've said, we use it at many places in the PAL and coreclr .